two_factor/views/core.py

```diff
@@ -540,6 +540,8 @@ def get_context_data(self, form, **kwargs):
             context.update({
                 'QR_URL': reverse(self.qrcode_url),
                 'secret_key': b32key,
+                'totp_digits': totp_digits(),
+                'issuer': get_current_site(self.request).name
```

Comment:
We should use `two_factor.utils.get_otpauth_url` to general this URL so that the otpauth URLs in the QR code match the URL in our HTML.

Reply:
@moggers87 Thanks for pointing out that function already existed!  Done. Ready for review again.